cypress/integration/pages/onDemandRadio/tests.js

```diff
@@ -23,13 +24,31 @@ export default ({ service, pageType, variant, isAmp }) => {
       });
     });
 
+    it('Brand page should not display a future episode', () => {
+      cy.request(
+        `${Cypress.env('currentPath')}.json${dataEndpointOverride()}`,
+      ).then(({ body: jsonData }) => {
+        const episodeAvailability = getEpisodeAvailability(jsonData);
+        if (episodeAvailability !== 'available') {
```

Comment:
This line is not needed anymore - just need to throw an error if it's a future episode, otherwise just log the availability